src/com/amazon/ionschema/IonSchemaSystemBuilder.kt

```diff
@@ -81,6 +82,21 @@ class IonSchemaSystemBuilder private constructor() {
         return this
     }
 
+    /**
+     * Allows top-level types to not have a name.  Such types can't be referred to
+     * by name and are thereby of limited (if any?) value.  This option if provided
+     * in case consumers have defined top-level types that don't have names.
+     * Should only be used if needed for backwards compatibility with v1.0; this will
+     * be removed in a future release.
+     *
+     * @since 1.1
+     */
+    @Deprecated("For backwards compatibility with v1.0")
+    fun allowUnnamedTopLevelTypes(): IonSchemaSystemBuilder {
```

Comment:
nit: unnamed = anonymous, rename to `allowAnonymousTopLevelTypes()`

Reply:
agreed, adds clarity, changed, thank you